data/presets/fields/piste_skitour/difficulty.json

```diff
@@ -0,0 +1,17 @@
+{
+    "key": "piste:difficulty",
+    "type": "combo",
+    "label": "Difficulty",
+    "placeholder": "Easy, Intermediate, Advanced...",
+    "strings": {
+        "options": {
+            "novice": "novice - L: inclination <30 degree",
+            "easy": "Easy - WS: inclination 30-35 degree",
+            "intermediate": "Intermediate - ZS: inclination 35-40 degree",
```

Comment:
I'll assume that these difficulty level descriptions are standard?

Reply:
I'd recommend using the degree symbol (°) instead of the word, to save space. Also, it might be nicer to use "30–35° incline" rather than "inclination 30-35 degree"

Reply:
Ive taken it from the wiki, will dig the history to check if it lies here long enough though. This come from a SAC scale.